lib/manageiq_performance/middlewares/active_record_queries.rb

```diff
@@ -72,7 +72,7 @@ def call(name, start_time, finish_time, id, payload)
 
               record_sql_query payload[:sql],
                                elapsed_time,
-                               binds_to_params(payload[:binds])
+                               payload[:binds]
```

Comment:
I like this being moved now to when we use it versus parsing before hand.  👍 

For the record, I was probably under some weird delusion thinking that passing in the full payload was going to cause memory bloat, even though I should know by now that Ruby is "pass by reference".